build_tools/waf_dynamo.py

```diff
@@ -1619,7 +1619,8 @@ def detect(conf):
         conf.env['STATICLIB_LUA'] = 'lua'
     else:
         conf.env['STATICLIB_LUA'] = 'luajit-5.1'
-        if '64' in build_util.get_target_platform():
```

Comment:
https://github.com/defold/defold/commit/91791a72b11eaa390e09451284937609b9d6f155#r62707167

Reply:
I'm thinking it's because the library wasn't rebuilt when we upgraded to latest luajit.
I'll try to rebuild it now.